scan-command/src/test/resources/test-resources/bal-project-with-platform-configurations/Scan.toml

```
@@ -0,0 +1,4 @@
+[[platform]]
+name = "examplePlatform"
+path = "C:\\Users\\Tharana Wanigaratne\\Desktop\\static-code-analysis-tool\\test-static-code-analysis-platform-plugins\\exampleOrg-static-code-analysis-platform-plugin\\build\\libs\\exampleOrg-static-code-analysis-platform-plugin.jar"
+key = "value"
```

```suggestion
key = "value"

```

Applicable elsewhere too.

---

Fixed with b83dd81b53a4df8201e6112720fbafc005dd9c27